indexer/map_object.cpp

```diff
@@ -136,6 +136,28 @@ std::string MapObject::GetLocalizedAllTypes(bool withMainType) const
   return oss.str();
 }
 
+std::string MapObject::GetAllReadableTypes() const
+{
+  ASSERT(!m_types.Empty(), ());
+  feature::TypesHolder copy(m_types);
+  copy.SortBySpec();
+
+  std::ostringstream oss;
+  bool isFirst = true;
+  for (auto const type : copy)
+  {
+    // Add fields separator between types
+    if (isFirst)
+      isFirst = false;
+    else
+      oss << feature::kFieldsSeparator;
+    
+    oss << classif().GetReadableObjectName(type);
```

Comment:
Since it is for debugging purpose, I'd prefer a simple:
```
oss << classif().GetReadableObjectName(type) << feature::kFieldsSeparator;
```
without any flags.

Reply:
Done